.github/workflows/ci_on_scratch.yml

```diff
@@ -0,0 +1,29 @@
+name: Normal CI
+
+on:
+  push:
+
+jobs:
+  run-test:
+    name: Run Test on Scratch
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+      - name: Install yarn
+        working-directory: ./frontend
+        run: npm install -g yarn
+      - name: Install yarn packages
+        working-directory: ./frontend
+        run: yarn install
+      - name: Install pip packages
+        working-directory: ./backend
+        run: python -m pip install -r requirements.txt
+      - name: pre commit run
+        uses: pre-commit/action@v2.0.3
+      - name: test frontend
+        working-directory: ./frontend
+        run: yarn test --watchAll=false
```

Comment:
```suggestion
        run: CI=true yarn test
```

Reply:
In CI, I'm actually assuming that CI option will be added.
https://github.com/ani-hovhannisyan/kanji-visualization/pull/23

However, `yarn test --watchAll=false` may be also OK.

Reply:
Sorry for missing your tips.
I'll update.